openmc/summary.py

```diff
@@ -23,13 +32,30 @@ def __init__(self, filename):
             raise ValueError(msg)
 
         self._f = h5py.File(filename, 'r')
-        self.openmc_geometry = None
-        self.opencg_geometry = None
+        self._openmc_geometry = None
+        self._opencg_geometry = None
 
         self._read_metadata()
         self._read_geometry()
         self._read_tallies()
 
+    @property
+    def openmc_geometry(self):
+        return self._openmc_geometry
+
+    @property
+    def opencg_geometry(self):
+        try:
+            from openmc.opencg_compatible import get_opencg_geometry
+        except ImportError:
+            msg = 'Unable to import OpenCG.'
```

Comment:
Without the try/except, wouldn't you get an almost equivalent message (No module named opencg)?